atlasdb-tests-shared/src/main/java/com/palantir/atlasdb/sweep/SweepTestUtils.java

```diff
@@ -70,7 +70,7 @@ public boolean isStandaloneServer() {
         ConflictDetectionManager cdm = ConflictDetectionManagers.createWithoutWarmingCache(kvs);
         Cleaner cleaner = new NoOpCleaner();
         LockAwareTransactionManager txManager = new SerializableTransactionManager(kvs, tsService, lockClient,
-                lockService, txService, constraints, cdm, ssm, cleaner, false);
+                lockService, txService, constraints, cdm, ssm, cleaner, false, 16, java.time.Duration.ofMinutes(1));
```

Comment:
Observation: it kinda sucks that you have to add these random parameters to all these constructors where you don't really care about range requests. Not sure what to do about it, though; tackling it is likely way beyond the scope of this already-very-large PR.

Reply:
Yeah agreed. Maybe can follow up with a wrapper that has some test defaults or something.